### PR TITLE
COMDOX-849: BIC highlights for 2.4.7 GA

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR):

- Adds a single BIC highlight for 2.4.7 GA (_New block class for subresource integrity verification_)
- Combines all previous 2.4.7-beta sections into a single section at the request of PM
- Alphabetizes the list of 2.4.7 BIC highlights

### TODO

- [x] Verify whether the `Magento_Elasticsearch7` module been removed for GA and update the related [highlight](https://developer.adobe.com/commerce/php/development/backward-incompatible-changes/highlights/#elasticsearch-7-deprecation) accordingly (e.g., remove references to 2.4.7-beta2)

## Review build

- https://adobedocs.github.io/commerce-php/development/backward-incompatible-changes/highlights/

### CODE REVIEW

File: src/pages/development/backward-incompatible-changes/highlights.md [hunk 1]
```diff
@@ -146,7 +101,32 @@ The following modules are affected by this change:
...
+All changes have been done in a way to minimize any impact to extensions and customizations. However, there are risks of conflict if an extension or customization extends the following:
+
+* `Magento\SalesRule\Model\ResourceModel\Rule\Collection::setValidationFilter` and adds a parameter to this method.
+* `Magento\SalesRule\Model\Validator` and introduces a method with the same name `initFromQuote`.
+
+The following module is affected by this change:
+
+* [Magento_SalesRule](https://developer.adobe.com/commerce/php/module-reference/module-sales-rule/)
+
+### New method for encyption key generation
```
Comment (jhadobe) [inline src/pages/development/backward-incompatible-changes/highlights.md:120]:
```suggestion
### New method for encryption key generation
```

File: src/pages/development/backward-incompatible-changes/highlights.md [hunk 2]
```diff
@@ -158,22 +138,41 @@ The following module is affected by this change:
...
 
-### `isEmailAvailable` API
+### New system configuration for limiting coupon generation
 
-The default behavior of the [`isEmailAvailable`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/queries/is-email-available/) GraphQL query and ([`V1/customers/isEmailAvailable`](https://adobe-commerce.redoc.ly/2.4.6-admin/tag/customersisEmailAvailable/#operation/PostV1CustomersIsEmailAvailable)) REST endpoint has changed. By default, the API now always returns `true`.
-The new default behaviour also affects the checkout workflow for guests that do not realize they already have an account. Previously, by default, when a guest supplied an email address that matched an existing customer account, they were prompted to sign in. Now, they are no longer prompted to sign in.
+Added a new setting for the number of coupons to generate. This property has a default value of `250,000`, which is also the maximum value. Merchants can disable this feature by it setting it to `0`  in the Admin by going to  **Stores** > **Settings** > **Configuration** > **Customers** > **Promotions** > **Code Quantity Limit**.
```
Comment (jhadobe) [inline src/pages/development/backward-incompatible-changes/highlights.md:161]:
```suggestion
Added a new setting for the number of coupons to generate. This property has a default value of `250,000`, which is also the maximum value. Merchants can disable this feature by setting it to `0`  in the Admin by going to  **Stores** > **Settings** > **Configuration** > **Customers** > **Promotions** > **Code Quantity Limit**.
```

File: src/pages/development/backward-incompatible-changes/highlights.md [hunk 3]
```diff
@@ -158,22 +138,41 @@ The following module is affected by this change:
...
 
-Merchants can restore the original default behavior of the `isEmailAvailable` API and checkout flow by setting the **Stores > Configuration > Sales > Checkout > Enable Guest Checkout Login field** to **Yes**. However, doing this can expose customer information to unauthenticated users.
+The following module is affected by this change:
+
+* [Magento_SalesRule](https://developer.adobe.com/commerce/php/module-reference/module-sales-rule/)
+
+### New system configuration for payment information rate limiting
+
+New native application rate-limiting features have been added with initial out-of-the-box support for rate limiting of payment API's. Disabled by default.
```
Comment (jhadobe) [inline src/pages/development/backward-incompatible-changes/highlights.md:169]:
```suggestion
New native application rate-limiting features have been added with initial out-of-the-box support for rate limiting of payment APIs. Disabled by default.
```

**Review by jhadobe — APPROVED** [review]

**Comment by jeff-matthews** [comment]
Broken internal link is expected. It will resolve after #245 is merged.

**Pull request merged**